microgen/rve.py

```diff
@@ -22,45 +22,52 @@ class Rve:
     :param dim: dimensions of the RVE
     """
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         dim_x: Optional[float] = None,
         dim_y: Optional[float] = None,
         dim_z: Optional[float] = None,
         center: Union[np.ndarray, Tuple, List] = (0, 0, 0),
-        dim: Optional[Union[float, np.ndarray, Tuple, List]] = 1,
+        dim: Union[float, np.ndarray, Tuple, List] = 1,
     ) -> None:
-        if isinstance(center, (Tuple, List)) and len(center) == _DIM:
+        if isinstance(center, (tuple, list)) and len(center) == _DIM:
```

Comment:
We are supposed to use `tuple` or `Tuple` ? to keep the compatibility with earlier python versions

Reply:
This is generally a problem with python 3.8, but mypy was showing a problem here and requesting this change. Tested with python 3.8 and it works fine.